tests/e2e/specs/modules/tagmanager/setup.test.js

```diff
@@ -169,4 +177,40 @@ describe( 'Tag Manager module setup', () => {
 		await expect( page ).toMatchElement( '.googlesitekit-setup-module--tag-manager .mdc-button', { text: /create an account/i } );
 		await expect( page ).toMatchElement( '.googlesitekit-setup-module--tag-manager .googlesitekit-cta-link', { text: /re-fetch my account/i } );
 	} );
+
+	describe.each(
+		Object.keys( allowedAmpModes )
+	)( 'AMP mode %s', ( mode ) => {
+		beforeEach( async () => {
+			await activatePlugin( 'amp' );
+			await activatePlugin( 'e2e-tests-module-setup-tagmanager-api-mock' );
+		} );
+		afterEach( async () => {
+			await deactivatePlugin( 'amp' );
+		} );
+		it( 'renders the correct drop downs in the setup screen', async () => {
+			await setAMPMode( mode );
+			await proceedToTagManagerSetup();
+
+			// Ensure that the correct dropdowns are displayed.
+			if ( 'standard' !== mode ) {
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--web' );
+			}
+			await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp' );
+		} );
+	} );
+	describe( 'Homepage AMP', () => {
```

Comment:
Let's integrate this with the above so that we check for valid AMP for both primary and secondary AMP setups.

Reply:
Updated

Reply:
This is done but I am getting failures locally. As we're making changes to the matchers, I'll mark this as resolved and figure those out as I move forward.